feature-detects/indexeddb.js

```diff
@@ -29,20 +29,26 @@ define(['Modernizr', 'prefixed', 'addTest'], function(Modernizr, prefixed, addTe
 
     if (!!indexeddb) {
       var testDBName = 'modernizr-' + Math.random();
-      var req = indexeddb.open(testDBName);
+      try {
+        // "indexeddb.open" may crash when used within an iframe
+        var req = indexeddb.open(testDBName);
 
-      req.onerror = function() {
-        if (req.error && req.error.name === 'InvalidStateError') {
-          addTest('indexeddb', false);
-        } else {
+        req.onerror = function(event) {
+          if (req.error && (req.error.name === 'InvalidStateError' || req.error.name === 'UnknownError')) {
```

Comment:
that part already got merged via https://github.com/Modernizr/Modernizr/commit/c5ffa86ccfa56a4a898b9004ea7a3aa337a09551

Reply:
so you can skip it in your merge